content/blog/2023-05-10-comparison-of-ethereum-classic-and-bitcoin-monetary-policies/index.md

```diff
@@ -0,0 +1,119 @@
+
+As said before, ETC started with no monetary policy other than a fixed payment per block. However, in 2017 it migrated to a schedule that is also “set in stone” as it establishes the whole future distribution in the software in the form of a supply algorithm very similar to that of Bitcoin.
+
+## Capped Supply
+
+Both ETC and BTC have a capped supply of their coins. This means that it’s known what is the total amount that will ever exist for each blockchain.
+
+Given the frequency of payments to miners, the number of coins issued per block, and its discount schedule, Bitcoin will have a maximum supply of 21,000,000 BTC.
+
+Given Ethereum Classic’s frequency of payments to miners, the number of coins it issues per block, and its discount schedule, it will have a maximum supply of ETC of 210,700,000.
+
+Note that Ethereum Classic’s maximum supply is similar to Bitcoin’s but 10 times higher. This was done purposefully to associate both cryptocurrencies reputationally and to make it easy for the market to understand ETC’s schedule.
+
+## Block Time
+
+Ethereum Classic’s block time is 13 seconds per block and Bitcoin’s block time is 10 minutes per block.
+
+This means that ETC miners create 6,646 blocks per day and BTC miners create 144 blocks in the same period.
+
+The block time of a blockchain is important because it establishes how fast the coins will be created, thus regulates the supply. 
+
+By modulating the block times and the number of coins created per block then a blockchain can control the monetary policy in time and supply levels. 
+
+## Eras
+
+Both Ethereum Classic and Bitcoin have eras measured in blocks in which the payments to miners, thus the issuance of coins, will be the same for each era.
+
+Ethereum Classic’s eras last 5 million blocks which, at 13 seconds per block, turns out to be more or less 2 years per era.
+
+Bitcoin’s eras last 210,000 blocks which, at 10 minutes per block, turns out to be more or less 4 years per era.
+
+After each era both ETC and BTC change or discount the payment to miners.
+
+## Discount Per Era
+
+In the case of Ethereum Classic, the discount per era is 20%. This means that since genesis, ETC has paid miners 5 ETC in the first era, then 4 ETC, then 3.20, and now is paying 2.56 which is the current era.
+
+Bitcoin’s discount per era is 50%. This means that since genesis, BTC has paid miners 50 BTC in the first era, then 25 BTC, then 12.50, and now is paying 6.25 which is the current era.
+
+## Pre-Mine
+
+Because Ethereum Classic and Ethereum were one single project since the beginning, and the project started with a crowd sale that resulted in an initial issuance, or what is called a “pre-mine”, then both ETC and ETH have a pre-mine stock of coins that were created at genesis block.
```

Comment:
Line 83: Rewrite for clarity and to remove compound sentence
```suggestion
Because Ethereum Classic and Ethereum were once a single project that started with the crowd sale of an initial issuance of coins called a “pre-mine”, both ETC and ETH have a pre-mined stock of coins that were created at the genesis block.
```